worldedit-core/src/main/java/com/fastasyncworldedit/core/configuration/Settings.java

```diff
@@ -192,7 +192,21 @@
                 "List of nbt tags to strip from blocks, e.g. Items",
         })
         public List<String> STRIP_NBT = new ArrayList<>();
-
+        @Comment({
+                "If the disallowed blocks listed in config-legacy.yml should be disallowed in all edits,",
+                "not just where blocks patterns are used.",
+                " - Can prevent blocks being pasted from clipboards, etc.",
+                " - If fast-placement is disabled, this may cause edits to be slower."
+        })
+        public boolean UNIVERSAL_DISALLOWED_BLOCKS = true;
+        @Comment({
+                "List of blocks to deny use of",
+        })
+        public List<String> DISALLOWED_BLOCKS = new ArrayList<>();
+        @Comment({
+                "List of blocks to strip nbt from",
+        })
+        public List<String> DISALLOWED_STATES = new ArrayList<>();
```

Comment:
Looking of the description of the option, wouldn't that be the same like `STRIP_NBT`?